pkg/api/api.go

```diff
@@ -77,6 +77,8 @@ type Config struct {
 	buildInfoEnabled bool `yaml:"build_info_enabled"`
 
 	QuerierDefaultCodec string `yaml:"querier_default_codec"`
+
+	DistributedExecEnabled bool `yaml:"distributed_exec_enabled" doc:"hidden"`
```

Comment:
This config should be defined in https://github.com/cortexproject/cortex/blob/master/pkg/querier/querier.go#L41 and pass down to NewQuerierHandler and so on